html/forms/input_elements.md

```diff
@@ -0,0 +1,44 @@
+# Input Elements and Form Tags Found in example-form.html
+
+## Different Input Elements
+
+There are input elements for the following pieces of data that the form collects:
+
+full name
+email
+password
+adventure name
+type of bicycle
+date
+consent
+
+## Different Form Tags and Their Purpose
+Input - to collect data from the user
+
+###Input attributes
+The type attribute specifies the style/format displayed to the user. It also does some validation to make sure what the user types in jives with the type. Here are the different types used in this file:
+- text
+- email
+- password
+- select element
+- date
+- checkbox
+- button
+
+The name attribute is used to identify the data when it is submitted.
```

Comment:
Hey lines 28-33 are not styled using markdown. can you please convert them into bullets?